### ISSUE TITLE
[A11y] The report dropdown menu does have the correct label for JAWS #3574

### ISSUE THREAD

**Issue by aliunwala** [IBMa/equal-access#960]
When a JAWS user navigates to the drop down report menu, the JAWS is reading as follows: 
`Report menu button menu`

JAWS user does not know that the button is closed 

- [ X] `a11y Checker`

## Environment

> Platform: Test
> Operating system: WIN
> Browser: Chrome
> Section:
> URL:
> WCAG 2.1 violation:  https://www.w3.org/TR/wai-aria-practices-1.2/#menubutton
> Assistive technology used to verify: JAWS

<img width="984" alt="Screen Shot 2021-09-20 at 1 33 45 PM" src="https://media.github.ibm.com/user/175242/files/913eec00-1a18-11ec-8d22-ababf9e99837">
<img width="1287" alt="Screen Shot 2021-09-20 at 12 25 06 PM" src="https://media.github.ibm.com/user/175242/files/92701900-1a18-11ec-8820-26d726b37827">

![image](https://user-images.githubusercontent.com/4451128/172458808-81162615-1cb9-41cc-90ee-b0d501a53461.png)

**Comment by aliunwala** [comment]
Triage: 
Should be a quick fix type bug. Seems like we just need to change the button text to just say "Report" and not "Report Menu"

**Comment by aliunwala** [comment]
Triage:
We spent a few minutes looking at this and this looks fixed. The reports icon now is labeled "Reports"
